### ISSUE TITLE
TipeeeStream doesn't work correctly

### ISSUE THREAD

**Issue by MelanX** [codeoverflow-org/chatoverflow#127]
### Description
Dev Branch
If you try to connect to tipeeestream you get an `xhr poll error`. @joblo2213 said it could be the changed port for the socket (`https://sso-cf.tipeeestream.com:443`) but it doesn't fix the problem for me.

### How to reproduce
1. Connect to TipeeeStream via ChatOverflow.
2. See the errors flow into your eyes.

### Error Log 
 
```
22888 12:21:46.847 [EventThread] WARN  tipeeestream.TipeeestreamConnector  - Could not connect to TipeeeStream socket:
22888 12:21:46.847 [EventThread] WARN  tipeeestream.TipeeestreamConnector  - io.socket.engineio.client.EngineIOException: xhr poll error
22982 12:21:46.941 [qtp1982787818-43] WARN  tipeeestream.TipeeestreamConnector  - Failed starting connector 'tipeee' of type 'org.codeoverflow.chatoverflow.requirement.service.tipeeestream.TipeeestreamConnector'.
22982 12:21:46.941 [qtp1982787818-43] WARN  instance.PluginInstance  - Failed to init connector (input) 'Tipeeestream Input' of type 'org.codeoverflow.chatoverflow.api.io.input.event.TipeeestreamEventInput'.
22983 12:21:46.942 [qtp1982787818-43] INFO  chat.TwitchChatConnector  - Unable to start connector 'twitch' of type 'org.codeoverflow.chatoverflow.requirement.service.twitch.chat.TwitchChatConnector'. Already running!
22983 12:21:46.942 [qtp1982787818-43] INFO  chat.TwitchChatConnector  - Instance count: 2
22983 12:21:46.942 [qtp1982787818-43] ERROR instance.PluginInstance  - At least one connector (input/output) did fail init. Unable to start.
23640 12:21:47.599 [EventThread] WARN  tipeeestream.TipeeestreamConnector  - Could not connect to TipeeeStream socket:
23640 12:21:47.599 [EventThread] WARN  tipeeestream.TipeeestreamConnector  - io.socket.engineio.client.EngineIOException: xhr poll error
```

**Comment by hlxid** [comment]
Passing the api key also with the query parameters like it says in the [tipeeestream guide](https://api.tipeeestream.com/api-doc/socketio) fixed this issue for me (ad0a16610e81940267092fde7260d51a6e349285). Judging based on the "Updated on August 7th, 2019." it is probably a new requirement that's required to connect.

@MelanX could you check if the problem is also fixed for you on current `develop`?

**Comment by hlxid** [comment]
@MelanX was able to report #132, meaning he was able to connect and authenticate and this bug is also fixed for him.
